src/server/hotkey_collector.cpp

```diff
@@ -28,10 +36,23 @@ void hotkey_collector::handle_rpc(const dsn::replication::detect_hotkey_request
 
 void hotkey_collector::capture_raw_key(const dsn::blob &raw_key, int64_t weight)
 {
-    // TODO: (Tangyanzhao) Add a judgment sentence to check if it is a raw key
+    dsn::blob temp_raw_key, hash_key, sort_key;
+    temp_raw_key = raw_key;
```

Comment:
`pegasus_restore_key ` will not modify the first parameter, i.e. `raw_key`, is `temp_raw_key` needed?